File: javascript/leap/leap.js
export const isLeap = (year) => {
  if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
      console.log("is leap year");
      return true;
  } else {
      console.log("is not leap year");
      return false;
  }
};

File: cpp/leap/leap.cpp
#include "leap.h"

namespace leap {
     bool is_leap_year(int year) {
        // Leap years are divisible by 4
        if (year % 4 == 0) {
            if (year % 100 == 0) {
                return year % 400 == 0;
            }
            return true;
        }
        return false;
    }
}



